src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -18,282 +21,295 @@ const intlWrapper = children => (
-      get: async () => ({
-        data: {
-          proctored_exam_settings: {
-            enable_proctored_exams: true,
-            allow_proctoring_opt_out: false,
-            proctoring_provider: 'mockproc',
-            proctoring_escalation_email: 'test@example.com',
-            create_zendesk_tickets: true,
-          },
-          available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
-          course_start_date: '2070-01-01T00:00:00Z',
-        },
-      }),
-    }));
-
-    auth.getAuthenticatedUser = jest.fn(() => ({ userId: 3, administrator: true }));
-    await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
-  });
-
+describe('ProctoredExamSettings', () => {
   afterEach(() => {
     cleanup();
   });
 
-  it('updates zendesk ticket field if proctortrack is provider', async () => {
-    await waitFor(() => {
-      screen.getByDisplayValue('mockproc');
-    });
-    const selectElement = screen.getByDisplayValue('mockproc');
-    await act(async () => {
-      fireEvent.change(selectElement, { target: { value: 'proctortrack' } });
-    });
-    const zendeskTicketInput = screen.getByTestId('createZendeskTicketsNo');
-    expect(zendeskTicketInput.checked).toEqual(true);
-  });
-
-  it('updates zendesk ticket field if software_secure is provider', async () => {
-    await waitFor(() => {
-      screen.getByDisplayValue('mockproc');
-    });
-    const selectElement = screen.getByDisplayValue('mockproc');
-    await act(async () => {
-      fireEvent.change(selectElement, { target: { value: 'software_secure' } });
-    });
-    const zendeskTicketInput = screen.getByTestId('createZendeskTicketsYes');
-    expect(zendeskTicketInput.checked).toEqual(true);
-  });
-
-  it('does not update zendesk ticket field for any other provider', async () => {
-    await waitFor(() => {
-      screen.getByDisplayValue('mockproc');
-    });
-    const selectElement = screen.getByDisplayValue('mockproc');
-    await act(async () => {
-      fireEvent.change(selectElement, { target: { value: 'mockproc' } });
-    });
-    const zendeskTicketInput = screen.getByTestId('createZendeskTicketsYes');
-    expect(zendeskTicketInput.checked).toEqual(true);
-  });
-
-  it('Hides all other fields when enabledProctorExam is false when first loaded', async () => {
-    cleanup();
-    auth.getAuthenticatedHttpClient = jest.fn(() => ({
-      get: async () => ({
-        data: {
-          proctored_exam_settings: {
-            enable_proctored_exams: false,
-            allow_proctoring_opt_out: false,
-            proctoring_provider: 'mockproc',
-            proctoring_escalation_email: 'test@example.com',
-            create_zendesk_tickets: true,
-          },
-          available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
-          course_start_date: '2070-01-01T00:00:00Z',
+  describe('Field dependencies', () => {
+    beforeEach(async () => {
```

Comment:
This first `beforeEach` here is representative of the changes throughout this file.  It uses `initializeMockApp` to set the authenticated user and spin up the mock services, then creates an axios MockAdapter instance, then sets up some mock requests using `onGet` or `onPost`.  

Later, we can run assertions against `axiosMock.history` to prove the requests went out as expected.